functests/1_performance/performance.go

```diff
@@ -1085,6 +1085,17 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			})
 		})
 	})
+
+	Context("Verify containers specifications", func() {
```

Comment:
I am not entirely sure this belongs to functional tests. But even if it does, I would expect comments or link to the bug so future readers understand WHY we have such a test in the first place.

Reply:
Sure, I will add a comment with a link to the bug. 
where do you think this tc should belong? 

thanks.

Reply:
Hello @MarSik , @fedepaol,
can you please review the PR?